R/expect-generic.R

```diff
@@ -1,31 +1,46 @@
 #' Expectations: generic expectation helpers
 #'
-#' These functions allow for testing of a dataset using an arbitrary function.
-#' Both `expect_all` and `expect_any` are wrappers around `expect_allany`. The
-#' former, `expect_all`, tests the variables in `vars` to see whether `func`
-#' returns TRUE for *all* of them (i.e. whether the conjunction of results of
-#' applying `func` to each of the `vars` is TRUE). The latter, `expect_any`,
-#' tests the `vars` to see whether `func` returns TRUE for *any* of them (i.e.
-#' whether the disjunction of the results of applying `func` to each of the
-#' `vars` is TRUE). The `expect_where` function works exactly like `expect_all`
-#' except that variables are specified not using `dplyr::vars()` (`vars`) but
-#' using bare [`tidy-select`][dplyr_tidy_select] functions (`where`).
+#' These functions allow for testing of multiple columns (`vars`) of a data
+#' frame (`data`), with an optional filter (`flt`), using an arbitrary function
+#' (`func`).
+#'
+#' * `expect_allany()` tests the columns in `vars` to see whether `func`
+#' returns `TRUE` for each of them, and combines the results for each row using
+#' the function in `allany`. Both `expect_all()` and `expect_any()` are wrappers
+#' around `expect_allany()`.
+#'
+#' * `expect_all()` tests the `vars` to see whether `func` returns `TRUE` for
+#' *all* of them (i.e. whether the conjunction of results of applying `func` to
+#' each of the `vars` is `TRUE`).
+#'
+#' * `expect_any()` tests the `vars` to see whether `func` returns `TRUE` for
+#' *any* of them (i.e. whether the disjunction of the results of applying `func`
+#' to each of the `vars` is `TRUE`).
+#'
+#' * `expect_where()` works exactly like `expect_all()`. When testdat used
```

Comment:
I haven't checked, but I assume this is redundant for the same reason `chk_filter_where()` is

Reply:
Yeah it just calls `chk_filter_where()` internally